db/types.go

```diff
@@ -0,0 +1,18 @@
+package db
+
+import (
+	"math/big"
+
+	"github.com/athanorlabs/atomic-swap/common/types"
+	contracts "github.com/athanorlabs/atomic-swap/ethereum"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+)
+
+// EthereumSwapInfo represents information required on the Ethereum side in case of recovery
+type EthereumSwapInfo struct {
+	StartNumber     *big.Int                  `json:"startNumber"`
```

Comment:
Can we document and, if appropriate, rename this field?  Is it the block number of the `newSwap` transaction? Is it the same for the maker and the taker? I wanted to rename it in `dimalinux/db-swap-info-json`, but I didn't know what it was without digging, so I left it.

Reply:
yeah it's the block number of the `newSwap`  tx, is the same for both, will update